packages/salesforcedx-vscode-apex/src/commands/forceLaunchApexReplayDebuggerWithCurrentFile.ts

```diff
@@ -81,13 +93,12 @@ async function getApexTestClassName(
   }
 
   await testOutlineProvider.refresh();
-  let testClassName = testOutlineProvider.getTestClassName(sourceUri);
-  // This is a little bizarre.  Intellisense is reporting that getTestClassName() returns a string,
-  // but it actually it returns string | undefined.  Well, regardless, since flushFilePath() takes
-  // a string (and guards against empty strings) using the Non-null assertion operator
-  // (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator)
-  // fixes the issue.
-  testClassName = fileUtils.flushFilePath(testClassName || '');
+
+  const filePath = sourceUri.toString();
+  // filePath is in the format of "file:///Users/{user-name}/{path-to-apex-file.cls}" on Mac OSX
+  // filePath is in the format of "file:///c%3A/Users/{user-name}/{path-to-apex-file.cls}" on Windows
```

Comment:
Randi ran this branch on her Windows machine this morning.  The code ran w/o any additional changes needed, and I added a comment here of what the format of `filePath` is here on Windows.